README.md

```diff
@@ -23,22 +105,20 @@ Once it's finished processing the default images (< 1 minute) you can then acces
 
 The Image Caption Generator endpoint must be available at `http://localhost:5000` for the web app to successfully start.
 
-### Configuring ports
+![Web UI Screenshot](assets/webui.png)
```

Comment:
we should move this file to `doc/source/images` since that's where readme images are stored in other code pattern repos

Reply:
Sure, I wasn't sure where to put it